nvm.sh

```diff
@@ -2449,7 +2460,7 @@ nvm() {
 
         # skip binary install if "nobinary" option specified.
         if [ $nobinary -ne 1 ] && nvm_binary_available "$VERSION"; then
-          if nvm_install_binary "${FLAVOR}" std "${VERSION}"; then
+          if NVM_NO_PROGRESS="${noprogress}" nvm_install_binary "${FLAVOR}" std "${VERSION}"; then
```

Comment:
Might I make a recommendation here as well:

```bash
if NVM_NO_PROGRESS="${NVM_NO_PROGRESS:-${noprogress}}" nvm_install_source ...
```

Why?  If you look up `Parameter Expansion` in the bash man page you'll find that `${VAR:-default}` will only take the default value if `$VAR` is unset.  The reason why this is useful is because it allows a user to set `NVM_NO_PROGRESS` globally as an environment variable on their system which will always disable progress without having to use the `nvm --no-progress` option.  In a CI-like, environment I feel this would be desirable.

Just my two cents on environment variables and using defaults.